plugins/modules/luks_device.py

```diff
@@ -300,6 +300,14 @@
         type: bool
         default: false
         version_added: '2.3.0'
+    allow_discards:
```

Comment:
You also need to update lines 296/297, since `allow_discards` can also be stored persistently.